src/resources/assets/js/volumes/geoFilter.js

```diff
@@ -1,33 +1,49 @@
 import {VolumeFilters} from './import';
 import {FilterList} from './import';
-
+import FilterSelect from '../geo/components/filterByLocationComponent';
 /**
  * Geo filter for the volume overview filters.
  */
+
 if (Array.isArray(VolumeFilters)) {
     VolumeFilters.push({
-        id: 'geo',
+        id: 'location',
         types: ['image'],
-        label: 'geo selection',
+        label: 'filter by location',
         help: "All images that were (not) selected on the world map.",
         listComponent: {
             mixins: [FilterList],
             data() {
                 return {
-                    name: 'geo selection',
+                    name: 'filter by location',
```

Comment:
I just noticed that the filter name does not match the naming scheme very well. In the active filter list, this will say "images with filter by location". Maybe call it "location selection" or even "geo selection" again? So it will say "images with geo selection".

Reply:
The filter name has been changed back to "geo selection" (85e7b2c009c1c6295de5226b58aa41533ccf275c)